contrib/current_search/current_search.module

```diff
@@ -212,6 +183,168 @@ function current_search_current_search_items() {
   );
 }
 
+/*
+ *
+ */
+function current_search_export_crud_load_all($table, $reset = FALSE) {
+  $schema = current_search_export_get_schema($table);
+  if (empty($schema['export'])) {
+    return array();
+  }
+  $export = $schema['export'];
+  if (!empty($export['load all callback']) && function_exists($export['load all callback'])) {
+    return $export['load all callback']($reset);
+  }
+  else {
+    return current_search_export_load_object($table);
+  }
+}
+
+/*
+ *
+ */
+function current_search_export_get_schema($table) {
```

Comment:
This is replacing `ctools_export_get_schema()`, which is still existing and called in the parent module. Why not just one `facetapi_get_schema()` and call that in both modules? Furthermore, since we're not actually using CTools, would the core function `backdrop_get_schema()` not suffice here? May not need all the CTools cruft.